docker-gc.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Docker-gc Service
+After=docker.service
+Requires=docker.service
+[Service]
+Restart=always
+ExecStartPre=-/usr/bin/docker kill docker-gc
+ExecStartPre=-/usr/bin/docker rm docker-gc
+ExecStartPre=/usr/bin/docker pull ndeloof/docker-gc:latest
```

Comment:
I'm curious: will docker-gc be able to correctly collect previous versions of itself when updated? (I admin this is not really related to this PR, but reading this line rang that bell)